GRG/js/drawGRG.js

```diff
@@ -667,6 +648,136 @@ define([
       callbackParamName: "callback"                          
     },{usePost: true});
     return def;
+  },
+  
+  grg.labelReferenceGrid = function (features, labelType, startPos, gridSize) {
+    var letterIndex = -1;
+    var number = 1;
+    var secondLetterIndex, letter,secondLetter;
+    var newLine = true;
+    var currentFeature = null;
+    var labeledFeatures = [];
+    
+    while (features.length > 0) {
+      if (newLine) {
+        letterIndex++;
+        if(labelType === 'alphaNumeric') {
```

Comment:
'alphaNumeric', 'alphaAlpha', etc... can all be encapsulated in an class level object for easier comparison.